lib/seattleflu/id3c/cli/command/etl/__init__.py

```diff
@@ -146,7 +146,7 @@ def ethnicity(ethnicity: str) -> Optional[bool]:
 
     >>> ethnicity("NOT HISPANIC OR LATINO")
     False
-    
```

Comment:
It looks like the only changes to this file are whitespace changes. Not a big deal, but since the file doesn't need to be modified here, its generally cleaner to exclude it and reset this file.